FiveCalls/FiveCalls/ScriptReplacements.swift

```diff
@@ -7,18 +7,46 @@
 //
 
 import Foundation
+import RegexBuilder
 
 struct ScriptReplacements {
     static func replacing(script: String, contact: Contact, location: NewUserLocation?) -> String {
-        var replacedScript = ScriptReplacements.replacingContact(script: script, contact: contact)
+        var replacedScript = ScriptReplacements.chooseSubscript(script: script, contact: contact)
+        
+        replacedScript = ScriptReplacements.replacingContact(script: replacedScript, contact: contact)
+        
         if let location {
             replacedScript = ScriptReplacements.replacingLocation(script: replacedScript, location: location)
         }
+        
         return replacedScript
     }
-
+    
+    static func chooseSubscript(script: String, contact: Contact) -> String {
+        let houseIntroPattern = /\*{2}WHEN CALLING HOUSE:\*{2}\n/
+        let senateIntroPattern = /\*{2}WHEN CALLING SENATE:\*{2}\n/
+        
+        func wholeRegex(_ introPattern: Regex<Substring>) -> Regex<Substring> {
+            return Regex {
+                introPattern
+                OneOrMore(.anyNonNewline)
+                OneOrMore(.newlineSequence)
+            }
+        }
+        
+        if contact.area == "US House" || contact.area == "House" {
+            var replacedScript = script.replacing(Regex(houseIntroPattern), with: "")
+            return replacedScript.replacing(wholeRegex(senateIntroPattern), with: "")
+        } else if contact.area == "US Senate" || contact.area == "Senate" {
+            var replacedScript =  script.replacing(Regex(senateIntroPattern), with: "")
+            return replacedScript.replacing(wholeRegex(houseIntroPattern), with: "")
+        }
+        
+        return script
+    }
+    
     static func replacingContact(script: String, contact: Contact) -> String {
-        let pattern = /\[REP\/SEN NAME\]|\[SENATOR\/REP NAME\]/
+        let pattern = /\[REP\/SEN NAME\]|\[SENATOR\/REP NAME\]|\[SENATOR NAME\]|\[REPRESENTATIVE NAME\]/
```

Comment:
Thanks for adding these extras, they really only come up in split scripts like this for clarity but we didn't handle replacements for them in the old app either